cartridge/confapplier.lua

```diff
@@ -632,12 +634,28 @@ local function init(opts)
     vars.advertise_uri = opts.advertise_uri
     vars.upgrade_schema = opts.upgrade_schema
 
-    local ok, err = remote_control.bind('0.0.0.0', vars.binary_port)
+    local parts = uri_tools.parse(opts.advertise_uri)
+    if not parts then
+        local err = InitError:new("Incorrect advertise uri %s", opts.advertise_uri)
```

Comment:
Already checked in https://github.com/tarantool/cartridge/blob/4323dffe75befb19c6e6a28788825e8e17848787/cartridge.lua#L466